helm/ako/values.yaml

```diff
@@ -40,11 +40,10 @@ NetworkSettings:
   #       - 11.0.0.1/24
   subnetIP: "" # Subnet IP of the vip network
   subnetPrefix: "" # Subnet Prefix of the vip network
-  networkName: "" # Network Name of the vip network
   enableRHI: false # This is a cluster wide setting for BGP peering.
-  vipNetworkList: []
```

Comment:
Can you change the comment, multiple networks are only supported for AWS cloud?

Reply:
We only support multi Vip for AWS added that in comment  in line no 44. Do you mean we can do this for other cloud too ?